parts/kubernetesmasteraddons-kube-proxy-daemonset.yaml

```diff
@@ -14,6 +14,11 @@ spec:
         component: kube-proxy
         tier: node
     spec:
+      tolerations:
```

Comment:
To enable daemonset/pod schedulable on master nodes, need to add the this piece into.